README.md

```diff
@@ -0,0 +1,42 @@
+# Metabase Node.js static embedding sample
+
+This sample code demonstrates using Metabase static embedding in Node.js using Express.
+
+# Running this sample
+
+## Pre-requisites
+* Have a running instance of Metabase. If you don't have one
+  * [download the free open source version](https://www.metabase.com/start/oss/) or
+  * [sign up for a free trial of Metabase Cloud](https://www.metabase.com/pricing/).
+* A dashboard to embed. If you don't have one, use X-Rays to let Metabase create one for you. Note down the dashboard id.
+
+## Preparing the embed
+1. Sign in to your Metabase instance as an admin. 
+2. Go to admin settings and enabling embedding.
+3. Under admin settings/embedding, click on static embedding and copy the embedding secret key.
+
+## Configure the app
+
+1. Paste the secret key into an env var: `$ export METABASE_EMBEDDING_SECRET="PASTE_SECRET_HERE"`
+2. Create an env var pointing to your Metabase site URL, if it's not on http://localhost:3000 `$ export METABASE_SITE_URL="http://localhost:4000"`
+3. Create an env var with the ID of the dashboard to embed: `$ export METABASE_EMBED_DASHBOARD_ID="8"`
+3. Install dependencies: `$ bundle install`
+
+## Embed the dashboard
+1. Go to your dashboard, click on the share/embed button at the top
+2. Click on the "Publish" button
+3. Install the dependencies: `$ npm install`
+4. Start the server: `$ node index.js`
+5. Open the app in your browser at http://localhost:8080
```

Comment:
Just tested locally, interestingly if you run Metabase on dev mode, our webpack dev server will run on a default port 8080 making this example not working. Not sure how big of a deal that is though, but what I had to change the example app to run on other port instead of 8080 to test with my Metabase instance on dev mode.

Reply:
This is mostly aimed at customers/users, so we can assume they are rather working with the downloaded JAR file or Docker image. I guess this is mostly an inconvenience internally, but I could still switch to any other port.

Do you have any suggestion for a particular number?

Reply:
Another whole number port could be a good option since they look nicer e.g. 4000, or 5000 though 4000 is used by Cypress, but I won't expect folks to run both Cypress and embedding at the same time so that might not be big of a deal.

Other options people used is to run the digits from the based port e.g. our dev server is running at 3000 and we could use 3001, 3002, 3003, etc. I have the [embedding app port set to 3001](https://github.com/WiNloSt/metabase-embeddings-101/blob/main/package.json#L6) with no particular reason.

tl;dr either 3xxx, or 4000 is fine IMO.